backend/src/Squidex/Program.cs

```diff
@@ -61,6 +63,16 @@ public static void Main(string[] args)
                 })
                 .ConfigureWebHostDefaults(builder =>
                 {
+                    builder.ConfigureKestrel((context, serverOptions) =>
+                    {
+                        if (context.Configuration.GetValue<bool>("devMode:enable"))
+                        {
+                            serverOptions.Listen(
+                                IPAddress.Any,
+                                context.Configuration.GetValue<int>("devMode:port"),
+                                listenOptions => listenOptions.UseHttps("squidex-dev.pfx", "password"));
```

Comment:
Is this a path or a cert name? If it is a path, it would not work

Reply:
this is not a path. Its Personal Information Exchange file which is created along with cert.

Reply:
But the docs say

```
//
        // Summary:
        //     Configure Kestrel to use HTTPS.
        //
        // Parameters:
        //   listenOptions:
        //     The Microsoft.AspNetCore.Server.Kestrel.Core.ListenOptions to configure.
        //
        //   fileName:
        //     The name of a certificate file, relative to the directory that contains the application
        //     content files.
        //
        //   password:
        //     The password required to access the X.509 certificate data.
        //
        // Returns:
        //     The Microsoft.AspNetCore.Server.Kestrel.Core.ListenOptions.
        public static ListenOptions UseHttps(this ListenOptions listenOptions, string fileName, string password);
```

Reply:
yes pfx file can be used as a cert file as well. It's kept under the same directory hence used directly with name. Have tested it and works like a charm